chatgpt_wrapper/gpt_shell.py

```diff
@@ -406,10 +407,36 @@ def do_ask(self, line):
         "`!ask` asks a question to chatgpt. It is purely optional. Example: `!ask what is 6+6` is the same as `what is 6+6`"
         return self.default(line)
 
+    def replace_file_strings(self, string):
+        # Regular expression to find all occurrences of !file and their arguments
+        pattern = r"!file\s+(\S+)"
```

Comment:
I'm wondering if we want a more strict format for doing this inline, for example:
```
{{ file:myfile.py }}
```
I'm not convinced that's the best, or even necessary, but I do suspect the current approach is too loose and error prone.

Part of why I'm suggesting this particular approach is that it would open us up to allow other kinds of simple token replacements like:
```
{{ stepwise }}
```
For the now classic "Let's think about this step by step"